ivy/functional/backends/numpy/elementwise.py

```diff
@@ -418,7 +418,10 @@ def log2(x: np.ndarray, /, *, out: Optional[np.ndarray] = None) -> np.ndarray:
 
 @_handle_0_dim_output
 def logaddexp(
-    x1: np.ndarray, x2: np.ndarray, /, *, out: Optional[np.ndarray] = None
```

Comment:
Please revert the change made here with the parameters.

Reply:
reverted the change